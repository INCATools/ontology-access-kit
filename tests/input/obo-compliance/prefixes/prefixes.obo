format-version: 1.4
ontology: prefixes
idspace: X http://example.org/X/

[Term]
id: X:1
is_a: Y:1

