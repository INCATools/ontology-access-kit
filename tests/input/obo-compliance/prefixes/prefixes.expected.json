{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/prefixes.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ],
    "edges" : [ {
      "sub" : "http://purl.obolibrary.org/obo/X_1",
      "pred" : "is_a",
      "obj" : "http://purl.obolibrary.org/obo/Y_1"
    } ]
  } ]
}